### ISSUE TITLE
On mobile, search results are broken, displaying blank images

### ISSUE THREAD

**Issue by chausies** [tagspaces/tagspaces#1657]
**Describe the bug**
On my Pixel 4 XL (running Android and the latest v4.4.3 apk of TagSpaces), search results are completely broken. Images show up fine when standard exploring, but as soon as I search, all photos show up completely blank in the results. What's more, when I click on them in the results, no image shows up (completely blank) and all the extra functions (like "Open Externally") don't work at all (they do nothing when I tap them). See the screenshot for details. I've tried uninstalling and re-installing multiple times. Each time, I delete the .ts folder just to make sure it's completely clean. But the problem still persists. I don't have this problem at all on Windows. Only on my Android phone.

**Screenshots**
Exploring the folder is fine:
![Screenshot_20220525-165716](https://user-images.githubusercontent.com/2512400/170388505-841fdf5b-f892-478f-8d7c-58a53af021b7.png)
Search results yield broken blank images:
![Screenshot_20220525-165752](https://user-images.githubusercontent.com/2512400/170388524-84031427-149f-4baa-8d70-97b94a06cca6.png)

**Additional context**
I have things set to sidecar mode, and have thumbnail generation enabled.

**Comment by Netmedia1** [comment]
The same  issue on Galaxy S8

**Comment by aisbergde** [comment]
same issue on all my android devices (smartphones, tablets, different Android versions)

**Comment by aisbergde** [comment]
I tried v5.0.3, but nothing is fixed. It is still not possible to use the app on android, if the content of the search results is not shown.

**Comment by sytolk** [comment]
![Screenshot_2022-10-18-09-39-29-968_org tagspaces mobileapp](https://user-images.githubusercontent.com/1415463/196354986-98e650a7-021b-4c95-87f6-29f528979920.jpg)

**Comment by aisbergde** [comment]
![Screenshot_2022-10-18-08-44-50-094_org.tagspaces.mobileapp.jpg](https://user-images.githubusercontent.com/9020187/196355651-2e818379-d02c-428e-9a45-bc54d0d7faf1.jpg)

![Screenshot_2022-10-18-08-44-59-577_org.tagspaces.mobileapp.jpg](https://user-images.githubusercontent.com/9020187/196355665-e360cd80-d736-4ee7-8adb-d4b1a71d4988.jpg)

but the content is shown, if I open the file from the unfiltered file list

btw, here you see another bug, sometimes the area, where sometimes they keyboard is located, is missing.
![Screenshot_2022-10-18-08-48-15-312_org.tagspaces.mobileapp.jpg](https://user-images.githubusercontent.com/9020187/196356378-69788951-d32e-470d-88ad-f26ca4e9fe8f.jpg)

but now the content is shown
![Screenshot_2022-10-18-08-50-24-076_org.tagspaces.mobileapp.jpg](https://user-images.githubusercontent.com/9020187/196356587-028acc5b-10d7-4d12-810a-1cc72cc86132.jpg)

**Comment by tagspaces-amp** [comment]
@aisbergde can you please tell us which viewer is configured for viewing the adoc files in the settings of the app? We have fixed a issue with the search index on Android. Using the old index could also cause this problem, you can try to regenerate the search index with the "Force Re-indexing" option in the advanced setting. Or simple deleting the tsi.json from the .ts folder of the location with the adoc files.

**Comment by aisbergde** [comment]
- I miss the option "Force Re-indexing" in the Advanced settings.
- I deleted tsi.json
- the issue is not related to .adoc files, but it happens also with other files, like .md, .txt
- adoc are configured:
  - File Opener -Text Editor
  - File Editor - TExt Editor
- md Files are configures
  - File Opener - MarkDown Editor
  - File Editor - MarkDown Editor

I am sure, it doesn't depend on the configured viewer or editor, but it looks like the files are differently called, when they are in the filtered list (filtered by tag) or when they are not filtered.

Because: in the filtered list, the entries in the context menu doesn't allow file operations

- "open File" - nothing happens
- "open file Natively" - I am asked, I am asked to confirm, if I really want to open "...", but pressing "OK" nothing happens
- "show in File Manager" - nothing happens
- "open Parent Folder" - I get a "S" in the upper left corner, and in the middle I get "No file or folder found"

But the same context menu items are partially working, if I apply them to a file in the not filtered list.

- "open File" - File opens
- "open file Natively" - I am asked, I am asked to confirm, if I really want to open "...", and pressing "OK", I am asked for the app to use, and I can open
- "show in File Manager" - nothing happens
- "open Parent Folder" - it works, and then it stays in the current folder.
- because I am not able to navigate into sub folders, I can't test, what would happen, if I would be in a sub folder

This means it basically has something to do with how the tag filtered files are accessed. Their filename is listed, access is not possible.

It is very hard to get the context menu. Sometimes I get it, sometimes not. It is clear that the app was not developed for Android, but is just somehow an adaptation of a desktop app (or something else) to Android. 

In general, it is very difficult to work meaningfully in the Android app:

- The context menu sometimes appears, sometimes not
- You can't navigate into sub folders
- the functions for short press, long press, double press are not intuitive, it's also hard to find useful settings in "Settings"

So basically I would say that the Android version is not usable. But I test it occasionally with new versions, hoping that it will become usable at some point.

Maybe this feedback here will help to make the Android version usable.

Android 12
MIUI Global 13.0.1

**Comment by uggrock** [comment]
Closing because the issue was fixed in #1731 , fix will be available in v5.1.4.
